eclass/git-r3.eclass

```diff
@@ -10,6 +10,21 @@
 # Third generation eclass for easing maintenance of live ebuilds using
 # git as remote repository.
 
+# @ECLASS_VARIABLE: EGIT_LFS
+# @PRE_INHERIT
+# @DESCRIPTION:
+# If set to 'yes' git lfs support will be enabled.
+# Set before inheriting this eclass.
+: "${EGIT_LFS:="no"}"
```

Comment:
Just make it `@DEFAULT_UNSET`. The eclass uses "non-null as enabled" logic elsewhere.